core/include/mm/fobj.h

```diff
@@ -112,6 +114,29 @@ static inline TEE_Result fobj_save_page(struct fobj *fobj,
 }
 #endif
 
+/*
+ * fobj_ta_mem_alloc() - Allocates TA memory
+ * @num_pages:	Number of pages
+ *
+ * If paging of user TAs read/write paged fobj is allocated otherwise a
+ * fobj which uses unpaged secure memory directly.
+ *
+ * Returns a valid pointer on success or NULL on failure.
+ */
+#ifdef CFG_PAGED_USER_TA
+#define fobj_ta_mem_alloc(num_pages) fobj_rw_paged_alloc((num_pages))
```

Comment:
Tab between macro and its value. Maybe s/`((num_pages))`/`(num_pages)`/.

Comment applies also to line 137.